hc/settings.py

```diff
@@ -126,6 +130,15 @@
 
 EMAIL_BACKEND = "djmail.backends.default.EmailBackend"
```

Comment:
Line 131 and 135 have a variable named the same.

Reply:
I've sorted this out. Thanks for pointing this out